Create Kolab dulu guis

erimm0 opened the pull request:
Bg izin pull request lagi, terserah mau di-acc atau ngga

erimm0 committed:
Create Kolab dulu guis

nendyrafi0 commented on Kolab dulu guis:
aneh banget anjay

nendyrafi0 reviewed (changes requested):
coba ganti nama file sama isinya dong, yang rada bener

nendyrafi0 reviewed (approved):
Oke sip

erimm0 committed:
Update and rename file

rename kolab dulu guis to collaboration, and change the text into something proper.

erimm0 committed:
Merge branch 'nendyrafi0:main' into main
